lib/vagrant-libvirt/action/start_domain.rb

```diff
@@ -124,13 +128,6 @@ def call(env)
                     cpu.delete_element(svm_feature)
                   end
                 end
-              else
-                unless cpu.elements.to_a.empty?
```

Comment:
Should this not be conditional on numa, e.g. do not skip this block `if  config.numa_nodes == nil`?